roles/heat/tasks/main.yml

```diff
@@ -54,6 +54,17 @@
     - heat-engine
   when: os == 'rhel' and openstack_install_method == 'source'
 
+- name: install heat services (rhel source)
+  systemd_service: name="openstack-{{ item }}"
+                   user=heat
+                   cmd=/usr/bin/{{ item }}
+                   config_files=/etc/heat/heat.conf
+  with_items:
+    - heat-api
```

Comment:
Services on rhel would be names
openstack-heat-api-cfn.service
openstack-heat-api.service
openstack-heat-engine.service